### DISCUSSION TITLE
ddb插件开发

### DISCUSSION THREAD

**Discussion by chenkangcrack** [dolphindb/api-cplusplus#13] [General]
想问下，官方可以写个详细的ddb用c++进行插件开发的案例吗？特别是c++中涉及到多线程，异步回调等，如何跟ddb线程进行通信，如果能写个ctp柜台封装的案例就更好了

**Comment by peeyee** [comment]
可以阅读以下文章：
https://gitee.com/dolphindb/Tutorials_CN/blob/master/plugin_development_tutorial.md
https://gitee.com/dolphindb/Tutorials_CN/blob/master/plugin_advance.md

另外，请前往 https://github.com/dolphindb/DolphinDBPlugin 提问插件开发相关问题。
